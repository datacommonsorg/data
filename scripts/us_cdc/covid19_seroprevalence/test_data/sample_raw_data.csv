Site,Date Range of Specimen Collection,Round,Catchment FIPS Code Description,Catchment Area Description,Catchment population,n [0-17 Years Prevalence],Rate (%) [0-17 Years Prevalence],Lower CI [0-17 Years Prevalence],Upper CI [0-17 Years Prevalence],CI Flag [0-17 Years Prevalence],n [18-49 Years Prevalence],Rate (%) [18-49 Years Prevalence],Lower CI [18-49 Years Prevalence],Upper CI [18-49 Years Prevalence],CI Flag [18-49 Years Prevalence],n [50-64 Years Prevalence],Rate (%) [50-64 Years Prevalence],Lower CI [50-64 Years Prevalence],Upper CI [50-64 Years Prevalence],CI Flag [50-64 Years Prevalence],n [65+ Years Prevalence],Rate (%) [65+ Years Prevalence],Lower CI [65+ Years Prevalence],Upper CI [65+ Years Prevalence],CI Flag [65+ Years Prevalence],n [Male Prevalence],Rate (%) [Male Prevalence],Lower CI [Male Prevalence],Upper CI [Male Prevalence],CI Flag [Male Prevalence],n [Female Prevalence],Rate (%) [Female Prevalence],Lower CI [Female Prevalence],Upper CI [Female Prevalence],CI Flag [Female Prevalence],n [Cumulative Prevalence],Rate (%) [Cumulative Prevalence],Lower CI [Cumulative Prevalence],Upper CI [Cumulative Prevalence],CI Flag [Cumulative Prevalence],Estimated cumulative infections count,Estimated cumulative infections lower CI 7,Estimated cumulative infections upper CI 7,All age-sex strata had at least one positive,Site Large CI Flag
AK,"Aug 6 - Aug 11, 2020",1,Statewide,Statewide,738516,5,777,,,,81,0.7,0,2.23,1,83,0,0,4.35,2,73,777,,,,88,0,0,4.11,2,154,0.7,0,2.35,1,242,0.3,0,1.12,1,2216,0,8271,false,true
AL,"Jul 29 - Aug 13, 2020",1,Statewide,Statewide,4864680,192,10.2,5.61,14.83,0,242,4.7,1.86,7.8,0,245,5.9,2.71,10,0,255,2.3,0.6,4.63,0,373,4.9,2.5,7.97,0,565,6.6,4.39,9.15,0,938,5.8,4.16,7.71,0,282151,202371,375067,true,false
AR,"Jul 29 - Aug 13, 2020",1,Statewide,Statewide,2990671,199,5,2,8.6,0,241,4.7,2.17,7.88,0,242,4.2,1.38,7.5,0,242,1.5,0,3.29,0,380,3.8,1.86,6.1,0,557,4.3,2.4,6.68,0,937,4.1,2.74,5.63,0,122618,81944,168375,true,false
AZ,"Jul 31 - Aug 11, 2020",1,Statewide,Statewide,6946685,24,777,,,,244,10.8,3.61,20.32,0,222,7.1,0.74,14.83,1,101,5.5,0.18,12.19,1,93,9,2.2,17.7,0,498,7.5,3.26,13.56,0,591,8.2,4.11,13.59,0,569628,285509,944054,true,true
CA,"Jul 30 -  Aug 5, 2020",1,Statewide,Statewide,39148760,245,6.2,3.12,10.09,0,245,7,3.9,10.3,0,245,5.2,2.46,8.82,0,245,1.1,0,2.61,1,420,5.1,2.74,7.83,0,560,6.3,3.97,9.19,0,980,5.7,4.05,7.77,0,2231479,1585525,3041859,true,true
CO,"Jul 30 -  Aug 7, 2020",1,Statewide,Statewide,5531141,240,4.3,0.63,10.52,1,246,1.1,0,2.75,1,258,4.3,0.34,11.31,1,257,0.8,0,2.07,1,454,2.7,0.75,5.72,0,547,2.1,0.49,4.94,0,1001,2.4,0.89,4.53,0,132747,49227,250561,true,true
CT,"Jul 30 -  Aug 3, 2020",1,Statewide,Statewide,3581504,248,3.9,1.48,6.46,0,249,3.1,1,5.62,0,248,3.7,1.3,6.3,0,249,2.7,0.5,5.17,0,417,4.2,2.13,6.42,0,577,2.5,1.18,4.1,0,994,3.4,2.04,4.72,0,121771,73063,169047,true,false
DC,"Jul 30 - Aug 13, 2020",1,Statewide,Statewide,684498,49,777,,,,262,1.4,0,3.09,1,185,5.5,2.28,9.15,0,219,6.4,3.28,9.94,0,321,3.3,1.27,5.93,0,396,4.4,2.06,7.53,0,717,3.9,2.18,5.76,0,26695,14922,39427,true,true
DE,"Jul 29 - Aug 13, 2020",1,Statewide,Statewide,949495,52,777,,,,237,8.3,4.31,12.72,0,261,5,2.24,8.34,0,254,0.9,0,2.29,1,339,4.3,2.13,7.12,0,465,7,4.02,10.92,0,804,5.7,3.55,8.22,0,54121,33707,78048,true,true
