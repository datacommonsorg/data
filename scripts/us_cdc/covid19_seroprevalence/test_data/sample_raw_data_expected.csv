Site,Count_0_17,Percent_0_17,LC_Percent_0_17,UC_Percent_0_17,Count_18_49,Percent_18_49,LC_Percent_18_49,UC_Percent_18_49,Count_50_64,Percent_50_64,LC_Percent_50_64,UC_Percent_50_64,Count_65,Percent_65,LC_Percent_65,UC_Percent_65,Count_Male,Percent_Male,LC_Percent_Male,UC_Percent_Male,Count_Female,Percent_Female,LC_Percent_Female,UC_Percent_Female,Count_Cumulative,Percent_Cumulative,LC_Percent_Cumulative,UC_Percent_Cumulative,Count_Cumulative_Infections,LC_Count_Cumulative_Infections,UC_Count_Cumulative_Infections,End_Date,Period
dcid:geoId/02,5,,,,81,0.7,0.0,2.23,83,0.0,0.0,4.35,73,,,,88,0.0,0.0,4.11,154,0.7,0.0,2.35,242,0.3,0.0,1.12,2216,0,8271,2020-08-11,P5D
dcid:geoId/01,192,10.2,5.61,14.83,242,4.7,1.86,7.8,245,5.9,2.71,10.0,255,2.3,0.6,4.63,373,4.9,2.5,7.97,565,6.6,4.39,9.15,938,5.8,4.16,7.71,282151,202371,375067,2020-08-13,P15D
dcid:geoId/05,199,5.0,2.0,8.6,241,4.7,2.17,7.88,242,4.2,1.38,7.5,242,1.5,0.0,3.29,380,3.8,1.86,6.1,557,4.3,2.4,6.68,937,4.1,2.74,5.63,122618,81944,168375,2020-08-13,P15D
dcid:geoId/04,24,,,,244,10.8,3.61,20.32,222,7.1,0.74,14.83,101,5.5,0.18,12.19,93,9.0,2.2,17.7,498,7.5,3.26,13.56,591,8.2,4.11,13.59,569628,285509,944054,2020-08-11,P11D
dcid:geoId/06,245,6.2,3.12,10.09,245,7.0,3.9,10.3,245,5.2,2.46,8.82,245,1.1,0.0,2.61,420,5.1,2.74,7.83,560,6.3,3.97,9.19,980,5.7,4.05,7.77,2231479,1585525,3041859,2020-08-05,P6D
dcid:geoId/08,240,4.3,0.63,10.52,246,1.1,0.0,2.75,258,4.3,0.34,11.31,257,0.8,0.0,2.07,454,2.7,0.75,5.72,547,2.1,0.49,4.94,1001,2.4,0.89,4.53,132747,49227,250561,2020-08-07,P8D
dcid:geoId/09,248,3.9,1.48,6.46,249,3.1,1.0,5.62,248,3.7,1.3,6.3,249,2.7,0.5,5.17,417,4.2,2.13,6.42,577,2.5,1.18,4.1,994,3.4,2.04,4.72,121771,73063,169047,2020-08-03,P4D
dcid:geoId/11,49,,,,262,1.4,0.0,3.09,185,5.5,2.28,9.15,219,6.4,3.28,9.94,321,3.3,1.27,5.93,396,4.4,2.06,7.53,717,3.9,2.18,5.76,26695,14922,39427,2020-08-13,P14D
dcid:geoId/10,52,,,,237,8.3,4.31,12.72,261,5.0,2.24,8.34,254,0.9,0.0,2.29,339,4.3,2.13,7.12,465,7.0,4.02,10.92,804,5.7,3.55,8.22,54121,33707,78048,2020-08-13,P15D
